courses/free-courses-hi.md

```diff
@@ -452,13 +452,15 @@
 
 ### Python
 
+* [100 Days of Code (Hindi) - Python Course](https://replit.com/learn/code-with-harry-100-doc) - replit
```

Comment:
Have you taken this course? I can't tell if it's real, or just a wrapper around Harry's courses

Reply:
Its real, best part is that you can code along with him in replit, so you don't have to switch tabs every time.

Reply:
Does Harry endorse it?

Reply:
Yep, cause his videos is done in replit. So it seems like they are in partnership with each other.

Reply:
ok

Reply:
wait, what do you mean "his videos is done in replit"???

Reply:
Like he is doing all his code in replit, and replit also has lessons in words. Basically, it's a great course, I've seen it and done a bit of it.

Reply:
sorry to belabor this, but should Harry be credited here?

Reply:
Ok, I credited both replit and CodeWithHarry